File: batches_c/batch_08_minor_logic_modification/compare.c
#include <stdio.h>

int max_of_two(int a, int b) {
    if (b <= a) return a;
    return b;
}

int main() {
    printf("%d\n%d\n", max_of_two(3, 7), max_of_two(10, 5));
    return 0;
}

File: batches/batches/batch_08_minor_logic_modification/compare.py
"""
Comparison - Minor Logic Modification (CHEATING)
Equivalent condition changes: a>b → b<=a, etc.
"""


def max_of_two(a, b):
    if b <= a:
        return a
    return b


if __name__ == "__main__":
    print(max_of_two(3, 7))
    print(max_of_two(10, 5))
